diff -ru i2b2Wizard/scripts/wizard_features.sh /opt/i2b2Wizard/scripts/wizard_features.sh
--- i2b2Wizard/scripts/wizard_features.sh	2015-04-15 11:10:23.000000000 +0000
+++ /opt/i2b2Wizard/scripts/wizard_features.sh	2015-04-15 11:06:25.000000000 +0000
@@ -215,7 +215,7 @@
     
 	cd $MY_PATH/packages
 	
-	if [ -f /etc/debian_version ] ; then 
+	if [ -f /etc/debian_version ] && [ $DBTYPE = "ORACLE" ] ; then 
 		INS=`aptitude search '~i ^oracle-xe-universal$'`
 		if ([ "$INS" = ""  ] && [ "$DB_SERVER" = "localhost" ] && [ -f /etc/debian_version ] ); then 
 			httpDownloadWizard oracle-xe-universal_10.2.0.1-1.1_i386.deb http://oss.oracle.com/debian/dists/unstable/non-free/binary-i386/oracle-xe-universal_10.2.0.1-1.1_i386.deb 3140db681260242c6e5951edfd5c17b5
diff -ru i2b2Wizard/wizard.sh /opt/i2b2Wizard/wizard.sh
--- i2b2Wizard/wizard.sh	2015-04-15 11:10:23.000000000 +0000
+++ /opt/i2b2Wizard/wizard.sh	2015-04-15 11:09:34.000000000 +0000
@@ -21,7 +21,7 @@
 DEFAULT_I2B2_VERSION=1.7.04
 DEFAULT_TRANSMART_VERSION=0
 
-DEFAULT_DB_TYPE=ORACLE
+DEFAULT_DB_TYPE=POSTGRESQL
 
 echo " +--------------------------------------------------------------------+  "
 echo " |                                                                    |  "
@@ -263,7 +263,7 @@
 PRODUCT_SCRIPT_IDENTIFIER="$PRODUCT"_"$PRODUCT_VERSION"
 
 while [ ! -f $MY_PATH/scripts/database_types/$DBTYPE.sh ]; do
-	echo -n "Please enter the database type that you intend to install (ORACLE (default), MSSQL or POSTGRESQL): "
+	echo -n "Please enter the database type you intend to use (ORACLE, MSSQL or POSTGRESQL) [$DEFAULT_DB_TYPE]: "
     read DBTYPE
 	FIRSTSTART=yes
 	
